app/controllers/pages_controller.rb

```diff
@@ -1,9 +1,17 @@
 # frozen_string_literal: true
 
 class PagesController < SecureController
+  def create
+    p = Page.create!(params.require(:data).require(:attributes)
```

Comment:
The reason I'm doing this is because with JSON api the request usually looks like this: